src/HoldRequestResultConsumerListener.php

```diff
@@ -309,6 +310,15 @@ protected function processListenerEvents()
                     APILogger::addDebug('Hold Request Id ' .
                         $holdRequestResult->getHoldRequestId() . ' is already processed.');
                 }
+            } catch (ClientTimeoutException $exception) {
+                APILogger::addError(
+                    'CurlTimedOutException thrown: ' . $exception->getMessage() .
+                    ', Error code: ' . $exception->getCode()
+                );
+                return new ListenerResult(
+                    false,
+                    'Retrying Timed Out Operation'
+                );
             } catch (RetryableException $exception) {
```

Comment:
I'm pretty sure you don't even need to catch this since you're `ClientTimeoutException` is extending `RetryableException`. You can just rely on the `RetryableException` catch if you want.